spec/integration/dashboard_integration_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+require "spec_helper"
+
+RSpec.describe DashboardController, type: :integration do
+  it "hits dashboard controller" do
+    user = User.create(first_name: "Mike", token: ENV['STRAVA_PERSONAL'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.